src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-alert */
+import React, { FC, useState } from 'react';
```

Comment:
`React` import not required

Expected:
`import { FC, useState } from 'react';`